packages/material-ui-lab/src/utils/helpers.js

```diff
@@ -0,0 +1,15 @@
+// It should to be noted that this function isn't equivalent to `text-transform: capitalize`.
+//
+// A strict capitalization should uppercase the first letter of each word a the sentence.
+// We only handle the first word.
+export function capitalize(string) {
+  if (process.env.NODE_ENV !== 'production' && typeof string !== 'string') {
+    throw new Error('Material-UI: capitalize(string) expects a string argument.');
+  }
+
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+export function contains(obj, pred) {
+  return Object.keys(pred).every(key => obj.hasOwnProperty(key) && obj[key] === pred[key]);
+}
```

Comment:
We can replace this with `import { capitalize } from '@material-ui/core/utils/helpers';`